new_platforms/docs/visualstudio_dev.md

```diff
@@ -42,48 +46,27 @@ flag when creating an enclave to automatically get the OP-TEE like behavior
 for both SGX and TrustZone.
 
 
-**Then to build for OP-TEE:**
+**Then to build the enclave for OP-TEE:**
 
-1. Right click the solution, select "Configuration Manager", create an ARM
-solution platform (if one doesn't already exist), and then create ARM
-configurations of your application project (and oehost if you include that
-directly).  Copy the configuration from the existing Win32 one.  Don't do
-this for your enclave project, as that needs to be built from a bash shell
-rather than in Visual Studio.
-2. Manually edit your application .vcxproj file to add the ability to
-compile for ARM, since Visual Studio cannot do it from the UI.  To do so, add the
-line "<WindowsSDKDesktopARMSupport\>true</WindowsSDKDesktopARMSupport\>"
-to each ARM configuration property group.  (See the sample apps'
-vcxproj file for examples.)
-3. For any new source files you add to your enclave project in
-Visual Studio, also add them to the sub.mk file in your optee subdirectory
-4. In your application project properties, under "Linker"->"Input", add
-rpcrt4.lib to the Additional Dependencies (All Configurations, ARM platform)
-which is required for string-to-UUID conversion, and remove any sgx libs.
-5. In your application project properties, update the Additional Include
-Directories to insert the $(NewPlatformsDir)include\optee\host and
-$(NewPlatformsDir)include\optee paths before the $(NewPlatformsDir)include path that you
-added earlier.
-6. On the destination machine, apply the uuids.reg file
+1. Using [Bash on Ubuntu on Windows](https://docs.microsoft.com/en-us/windows/wsl/about) shell, 
```

Comment:
Perhaps: "Using the [Bash on Ubuntu on Windows](...) shell [...]".

Reply:
done